### DISCUSSION TITLE
MusicXML file extension

### DISCUSSION THREAD

**Discussion by DavidMWebber** [w3c-cg/musicxml#464] [General]
Previously my experience has been that musicXML files have the extension .xml and compressed music XML is .mxl

However recently I was sent a file with extension .musicxml   (encased, as it happens, in a .mxl file).    It said it was encoded by Finale.

Is .musicxml regarded as a standard extension which my software (http://www.mozart.co.uk) needs to look for?  {When importing various formats it identifies them by extension .mid, .abc. .nif .xml, .mxl    Do I need to add .musicxml? )

Dave

**Comment by Jojo-Schmitz** [comment]
Yes. .musicxml is the same format as .xml: uncompressed MusicXML
MuseScore uses .musicxml since many years now, since 2.2, released back in March 2018. Finale does so since v25.4, June 2017.
I believe this was part of MusicXML 3.1.

**Reply by DavidMWebber** [reply]
Thanks.  Is .xmk a typo for .xml or is this yet another extension in use?

**Reply by Jojo-Schmitz** [reply]
Typo, sorry, corrected now

**Reply by DavidMWebber** [reply]
Thanks again!   I am surprised, from what you say, not to have encountered .musicxml before today, but I shall add it to my collection of recognised extensions.  (I had no problem opening it but had to rename it manually as .xml).

Dave

**Reply by samuelbradshaw** [reply]
See https://www.w3.org/2021/06/musicxml40/tutorial/compressed-mxl-files/#file-suffixes-and-media-types for more info.

**Reply by DavidMWebber** [reply]
Thanks - I hadn't been working on MusicXML aspects in the last year or so.  I obviously need to explore MusicXML 4 further!

**Reply by Jojo-Schmitz** [reply]
As mentioned: that .musicxml thing is from 3.1 already.

**Reply by DavidMWebber** [reply]
OK I need to explore 3.1 AND 4 a bit further :-)
